### ISSUE TITLE
Random changes of song rating

### ISSUE THREAD

**Issue by Aquan1412** [quodlibet/quodlibet#3516]
Hello, 
I filed this as bug report, even though I'm not completely sure if it is a bug, or maybe mishandling on my part. My problem is, that Quod Libet sometimes seemingly randomly changes the rating of songs. So far, I only noticed it with five-star ratings, probably because originally I only had very few of those.

What happens is, that  QL sets the rating of some previously unrated songs to five stars. Today it was especially noticeable, since I suddenly had dozens of new five-star ratings. In the settings I set QL to use a rating scale of 0 to 5 stars, with a default rating of 0 stars, so that shouldn't be the reason. Also, the changed songs weren't newly imported or anything, and I didn't see any obvious connection between them (they were from different albums, in some albums all songs were now rated 5 stars, in other albums it was only a couple of songs...).

I'm not able to reproduce the issue, so far it happened mybe three or four times, without any obvious trigger. 

I was wondering, is there maybe a shortcut to rate several songs at once? Maybe I accidentally used the shortcut to rate all songs that were returned by a search...

**Comment by declension** [comment]
Hi. Only a few things I can think of:

1.  Bulk rating (select many files, right click, Rating -> 5 stars), now requires a confirmation prompt though (so depends what version you're on but think it's been in ages now)
1. _Automatic Rating_ plugin - this does happen behind the scenes once enabled, so might be the most likely candidate unless it is a weird bug (that hasn't been reported so far)
1. _Set Exact Rating_ plugin, but pretty unlikely you won't remember doing that

Obviously hard to say any more without seeing some debug logs but even then might be tricky :confused:

**Comment by Aquan1412** [comment]
Thanks for the quick reply. I just checked, and I didn't activate any plugin related to ratings. I did, however, deactivate the confirmation for bulk changes of ratings...
So, I guess that was the problem. I re-activated the setting, so hopefully it won't be a problem anymore. However: is there a way to keep debug logs running all the time, so if the problem does persist, I might have something to follow up on?
